app/models/competition.rb

```diff
@@ -1933,10 +1914,10 @@ def persons_wcif(authorized: false)
       } },
       :wcif_extensions,
     ]
-    # V2 registrations store the event IDs in the microservice data, not in the monolith
-    includes_associations << :events unless self.uses_microservice_registrations?
 
-    registrations_relation = self.uses_microservice_registrations? ? self.microservice_registrations : self.registrations
+    includes_associations << :events
+
+    registrations_relation = self.registrations
```

Comment:
You don't need to explicitly declare this into a variable anymore, you can just use `self.registrations` directly at the call-site because (I think) it's only being used once.